apps/web/src/layouts/LayoutWrapper.tsx

```diff
@@ -45,6 +45,11 @@ export function LayoutWrapper({ children }: { children: ReactNode }) {
     if (routeChain) setChain(routeChain)
   }, [query.network, setChain])
 
+  // Fix for dev: network must be reset when env network type changes
+  React.useEffect(() => {
+    if (!query?.network) return setChain(PUBLIC_DEFAULT_CHAINS[0])
```

Comment:
if this is for dev only should we also check for the env type?

Reply:
good call just added that in @nadiakiamilev